frontend/src/app/candidates/candidate-list/candidate-form/candidate-form.component.css

```diff
@@ -0,0 +1,40 @@
+.candidateForm { 
```

Comment:
do not use camelCase in css selectors
Few links for reference:
http://bdavidxyz.com/blog/how-to-name-css-classes/
http://bdavidxyz.com/blog/how-to-name-css-classes/

Reply:
Thank you for information. I will correct.